### ISSUE TITLE
[typhoon] Error after turon LangSwitcher

### ISSUE THREAD

**Issue by PepaR** [getgrav/grav-premium-issues#200]
Hello, if I turn on LangSwitcher plugin, I get critical error (see below).
May you help me, how to solve it, please?

`[2022-01-09 16:40:13] grav.CRITICAL: An exception has been thrown during the rendering of a template ("Creating default object from empty value"). - Trace: #0 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(420): Twig\Template->displayWithErrorHandling(Array, Array) #1 /www/doc/www.stavlab.cz/www/cache/twig/e8/e8754f4b481b19e150f2bbae124dfcb16678bc4d6a701d20d2de2640555b28bf.php(76): Twig\Template->display(Array) #2 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(453): __TwigTemplate_2f1fa5b76967d6b8c6c7c964e6c7792e5698fdb9b0b873271c65b9756ba24829->doDisplay(Array, Array) #3 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(420): Twig\Template->displayWithErrorHandling(Array, Array) #4 /www/doc/www.stavlab.cz/www/cache/twig/34/343a2c7f5fe7c101df01e4f849bf33f3b49996163a5bad8dcae1d2dfdb1e38f9.php(250): Twig\Template->display(Array) #5 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(214): __TwigTemplate_9a414d0ccfcfc8f71e9b430514c6fe8522369bd227f6dfee93b85ed6c5edfc81->block_header(Array, Array) #6 /www/doc/www.stavlab.cz/www/cache/twig/34/343a2c7f5fe7c101df01e4f849bf33f3b49996163a5bad8dcae1d2dfdb1e38f9.php(121): Twig\Template->displayBlock('header', Array, Array) #7 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(214): __TwigTemplate_9a414d0ccfcfc8f71e9b430514c6fe8522369bd227f6dfee93b85ed6c5edfc81->block_render_body(Array, Array) #8 /www/doc/www.stavlab.cz/www/cache/twig/d8/d8cab77829a8ddecc427241d77ea6a1dafa0819953e1aeba988f7706338976db.php(55): Twig\Template->displayBlock('render_body', Array, Array) #9 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(453): __TwigTemplate_caab7fadac1fd4e454f378f1662d678c64863c5a436eb2ffe1ef4d5deb8d48c1->doDisplay(Array, Array) #10 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(420): Twig\Template->displayWithErrorHandling(Array, Array) #11 /www/doc/www.stavlab.cz/www/cache/twig/34/343a2c7f5fe7c101df01e4f849bf33f3b49996163a5bad8dcae1d2dfdb1e38f9.php(58): Twig\Template->display(Array, Array) #12 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(453): __TwigTemplate_9a414d0ccfcfc8f71e9b430514c6fe8522369bd227f6dfee93b85ed6c5edfc81->doDisplay(Array, Array) #13 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(420): Twig\Template->displayWithErrorHandling(Array, Array) #14 /www/doc/www.stavlab.cz/www/cache/twig/8e/8ebac86c82bce62b074e8eb29f6064ca3939ddb7514da0312b5cdf9ce11318cb.php(35): Twig\Template->display(Array, Array) #15 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(453): __TwigTemplate_6cb32a8ca61bcc1fc47fd00162b1e3fa3efd4b939025ffb996ac3f02d9741a7e->doDisplay(Array, Array) #16 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(420): Twig\Template->displayWithErrorHandling(Array, Array) #17 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Template.php(432): Twig\Template->display(Array) #18 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/TemplateWrapper.php(47): Twig\Template->render(Array, Array) #19 /www/doc/www.stavlab.cz/www/vendor/twig/twig/src/Environment.php(384): Twig\TemplateWrapper->render(Array) #20 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Twig/Twig.php(439): Twig\Environment->render('error.html.twig', Array) #21 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Service/OutputServiceProvider.php(36): Grav\Common\Twig\Twig->processSite('html') #22 /www/doc/www.stavlab.cz/www/vendor/pimple/pimple/src/Pimple/Container.php(122): Grav\Common\Service\OutputServiceProvider->Grav\Common\Service\{closure}(Object(Grav\Common\Grav)) #23 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/RenderProcessor.php(40): Pimple\Container->offsetGet('output') #24 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\RenderProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #25 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #26 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/DebuggerAssetsProcessor.php(38): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #27 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\DebuggerAssetsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #28 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #29 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/PagesProcessor.php(112): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #30 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\PagesProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #31 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #32 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/TwigProcessor.php(38): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #33 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\TwigProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #34 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #35 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/AssetsProcessor.php(39): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #36 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\AssetsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #37 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #38 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/SchedulerProcessor.php(40): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #39 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\SchedulerProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #40 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #41 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/BackupsProcessor.php(39): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #42 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\BackupsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #43 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #44 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/TasksProcessor.php(69): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #45 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\TasksProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #46 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #47 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/RequestProcessor.php(63): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #48 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\RequestProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #49 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #50 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/ThemesProcessor.php(38): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #51 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\ThemesProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #52 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #53 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/PluginsProcessor.php(39): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #54 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\PluginsProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #55 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #56 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/InitializeProcessor.php(130): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #57 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Debugger.php(546): Grav\Common\Processors\InitializeProcessor::Grav\Common\Processors\{closure}() #58 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Processors/InitializeProcessor.php(131): Grav\Common\Debugger->profile(Object(Closure)) #59 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(50): Grav\Common\Processors\InitializeProcessor->process(Object(Nyholm\Psr7\ServerRequest), Object(Grav\Framework\RequestHandler\RequestHandler)) #60 /www/doc/www.stavlab.cz/www/system/src/Grav/Framework/RequestHandler/Traits/RequestHandlerTrait.php(62): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #61 /www/doc/www.stavlab.cz/www/system/src/Grav/Common/Grav.php(306): Grav\Framework\RequestHandler\RequestHandler->handle(Object(Nyholm\Psr7\ServerRequest)) #62 /www/doc/www.stavlab.cz/www/index.php(54): Grav\Common\Grav->process() #63 {main} [] []`

**Comment by rhukster** [comment]
Do you have the langswitcher plugin installed?

**Comment by rhukster** [comment]
Also please make sure you have Grav and all plugins up to date.  I can't replicate this BTW, even if i disable langswitcher plugin. So i'll need more information about your setup.

**Comment by rhukster** [comment]
@PepaR ping :)

**Comment by w00fz** [comment]
Going to mark this issue as resolved due to inactivity. Feel free to comment on this anytime @PepaR, we can always reopen.

Cheers,

**Comment by PepaR** [comment]
Hello, I'm sorry for delay. Yes, I langswitcher plugin is installed.

**Comment by rhukster** [comment]
Just zip up and share the whole user/ folder.

**Comment by w00fz** [comment]
Don't upload it here on GitHub, share it in Discord please. You can DM the zip to either Andy or myself on Discord or send it via email at premium@getgrav.org .

Thanks!

**Comment by PepaR** [comment]
Thanks, I sent link to zipped user folder via Discord.

**Comment by rhukster** [comment]
so sorry @PepaR, i've been busy with client projects and only today have I been able to catch up on things.  

1. I've downloaded your zip, and installed in a latest Grav installation (1.7.30).
2. I extracted your zip and replaced the default user folder.
3. I created an admin account and logged in
4. I enabled lang-switcher plugin
5. I disabled maintenance plugin so I could see the frontend

No issues, front end loads and shows the langswitcher as expected:

<img width="511" alt="CleanShot 2022-02-25 at 18 51 47@2x" src="https://user-images.githubusercontent.com/1084697/155824424-aa006027-9f6b-4843-9cda-ec92dca06f70.png">

I'm running PHP 8.0 for the record.  nothing special really. I didn't touch anything else in your files.
